README.md

```diff
@@ -9,7 +9,7 @@ The p8e-cee-api allows for operations against the encrypted object store, with i
 [![License][license-badge]][license-url]
 [![LOC][loc-badge]][loc-report]
 
-[release-badge]: https://img.shields.io/github/v/tag/provenance-io/p8e-cee-api.svg?sort=semver
+[release-badge]: https://img.shields.io/github/v/tag/provenance-io/p8e-cee-api.svg
```

Comment:
Not that it matters, but if I had to guess, you deleted this purely because you CMD + F'ed "semver" while trying to remove references to the Gradle plugin 😛